src/block/carousel/frontend-carousel.js

```diff
@@ -2,6 +2,7 @@
  * WordPress dependencies
  */
 import domReady from '@wordpress/dom-ready'
+import { throttle } from 'lodash'
```

Comment:
We cannot use `throttle` here because we do not want to load `lodash` in the frontend, we'll need to just do this the `setTimeout` way